http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/auth/sam/SamAuthProvider.scala

```diff
@@ -72,12 +73,25 @@ class SamAuthProvider[F[_]: Effect: Logger](samDao: SamDAO[F],
         )
       )
 
+  private def getPersistentDiskActionString(action: LeoAuthAction): Either[Throwable, String] =
+    persistentDiskActionMap
+      .get(action)
+      .toRight(
+        UnknownLeoAuthAction(
+          s"SamAuthProvider has no mapping for persistent-disk authorization action ${action.toString}, and is therefore probably out of date."
+        )
+      )
+
   private val projectActionMap: Map[LeoAuthAction, String] = Map(
     GetClusterStatus -> "list_notebook_cluster",
     CreateClusters -> "launch_notebook_cluster",
     SyncDataToCluster -> "sync_notebook_cluster",
     DeleteCluster -> "delete_notebook_cluster",
-    StopStartCluster -> "stop_start_notebook_cluster"
+    StopStartCluster -> "stop_start_notebook_cluster",
+    CreatePersistentDisk -> "create_persistent_disk",
+    ModifyPersistentDisk -> "modify_persistent_disk",
```

Comment:
Didn't we get rid of `modify` at this project level… should this be `list_persistent_disk`?